modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -123,6 +138,14 @@ vec4 project_position(vec4 position, vec3 position64Low) {
       );
     }
   }
+  if (project_uProjectionMode == PROJECTION_MODE_GLOBE) {
```

Comment:
Nit: These nested if statements are getting pretty big, may read cleaner if we have helper functions `projectGlobe` `projectFlat` etc.